src/actions/image_agents.ts

```
@@ -89,8 +89,10 @@
     if (plugin.markdown) {
       returnValue.markdown = plugin.markdown({ beat, context, imagePath, ...htmlStyle(context, beat) });
     }
+
+    const isMovie = beat.image.type === "movie";
```

ESLint

[error] 'isMovie' is already declared in the upper scope on line 52 column 9.

(no-shadow)